### PR TITLE
feat(components): Add css changes to show the placeholder label to the form field [JOB-83555]

### PR DESCRIPTION
**Pull request by rahulshial**
## Motivations

The InputTime element was not showing the `Placeholder label` when the element was not focussed. SInce incorporating the new InputTime element for our reactified `Scherdule Card`, SPs have provided feedback regarding the InputTime element, stating that the field input seemed mandatory, which it was not. Hence there was a need to show the `Placeholder label` when not focussed.

### Added

- 

### Changed

- Added `timeInputLabel` to FormField.css.d.ts
- Updated FormField.css to show `placeholder label` when there is no value in the time field element
- make FormFieldProps `readonly`

## Testing
- [ ] To test this feature, run `npm start` to start `storybook`, navigate to `InputTime` from SearchBox, navigate to `Event`
image

- [ ] `Start Time` should be displayed in the input element
- [ ] Clicking in (focussing) the input time element should 
  - [ ] replace the `placeholder label` with the `time input`
  - [ ] move the `placeholder label`  to above the `time input`
- [ ] Clicking out (focus out) of the time input element should
  - [ ] show the `placeholder label` instead of the time input
- [ ] To test in `Safari` copy the `Storybook url` into `Safari` address bar, and you should be able to test the above steps
- [ ] To test in mobile / tablet (IOS) start `simulator`, and copy   the `Storybook url` into `Safari` address bar, and you should be able to test the above steps. `NOTE: Simulator will not allow deleting the time`
- [ ] To test locally with `Jobber Online`, go thru the process noted here -> [npm pack](https://atlantis.getjobber.com/?path=/docs/introduction--page#local-testing)

https://github.com/GetJobber/atlantis/assets/68708089/213aab1f-3d73-4b85-a5ac-925a4d49bc59

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/InputTime/Web.stories.tsx [hunk 1]
```diff
@@ -26,13 +26,16 @@ const BasicTemplate: ComponentStory<typeof InputTime> = args => (
 );
 
 const EventTemplate: ComponentStory<typeof InputTime> = args => {
-  const [time, setTime] = useState(new CivilTime(3, 52));
+  const [time, setTime] = useState<CivilTime>();
+
   const resetTime = () => {
     setTime(new CivilTime(3, 52));
```
Comment (rinheimerle) [inline docs/components/InputTime/Web.stories.tsx:32]: I wonder since the default is no longer 3, 52 if we should change this to reflect that?

**Review by rinheimerle — COMMENTED** [review]
QA steps:

1. Checked out on Chrome, Safari and Firefox (Firefox wasn't required but I thought I'd check it anyway)
2. Checked out on mobile as well
3. Tested using the `npm pack` in Jobber Online
4. Clicked into the field
5. Tabbed into the field
6. Each time the start time label acted accordingly (moved to above the time input) and replaced the value. 

I had one question about the reset time value, once addressed this should be good to merge.

File: packages/components/src/FormField/FormFieldWrapper.tsx [hunk 2]
```diff
@@ -50,14 +50,15 @@ export function FormFieldWrapper({
...
-        (placeholder && type === "time") ||
         // Naively assume that if the the type is tel, it is the InputPhoneNumber
         (placeholder && type === "tel"),
       [styles.textarea]: type === "textarea",
       [styles.select]: type === "select",
       [styles.invalid]: invalid ?? error,
       [styles.disabled]: disabled,
       [styles.maxLength]: maxLength,
+      [styles.timeInputLabel]:
+        placeholder && type === "time" && placeholder && value === "",
```
Comment (darryltec) [inline packages/components/src/FormField/FormFieldWrapper.tsx:61]: can you add a test for this? Make sure it only gets added when it needs to be.

File: docs/components/InputTime/Web.stories.tsx (hunk 1 above)
Comment (rahulshial) [reply docs/components/InputTime/Web.stories.tsx:32]: Yep. Done. This will show up in storybook.

**Review by rinheimerle — APPROVED** [review]
LGTM QA + CR!

**Pull request merged**